src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderProfileDetail.java

```diff
@@ -53,10 +53,10 @@ public static class PK implements Serializable {
         private Integer sequence;
     }
 
-    @ManyToOne(optional = false)
```

Comment:
the code is indeed optional, and typically not present for rows created by NOMIS